labs/02_cli.md

```diff
@@ -4,17 +4,39 @@ In diesem Lab werden wir gemeinsam das CLI-Tool `odo` installieren und konfiguri
 
 ## `oc`
 
-FIXME: Anleitung für Installation von `oc` zur Verfügung stellen, `odo` entfernen.
+Der __oc client__ stellt ein Interface zu OpenShift bereit.
+
+### Installation
+
+Analog der [offiziellen Dokumentation](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands) kann der __oc client__ von der __Infrastructure Provider__ Seite heruntergeladen werden.
+
+__Tipp__:
+Alternativ kann die Binary auch direkt mittels folgenden Befehlen im Terminal installiert werden:
+
+```
+curl -fsSL https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.2/linux/oc.tar.gz | sudo tar xfz - -C /usr/bin
+```
+
+### bash Command Completion (optional)
+
+Dieser Schritt ist optional und funktioniert nicht auf Windows. Damit Command Completion auf macOS funktioniert, muss bspw. via `brew` das Paket `bash-completion` installiert werden.
+
+`oc` bietet eine Command Completion, die gem. [Dokumention](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/configuring-cli.html#cli-enabling-tab-completion_cli-configuring-cli) eingerichtet werden kann.
+
+__Tipp__:
+Alternativ kann die Bash Command Completion auch mittels folgenden Befehlen im Terminal installiert werden:
+
+```
+sudo oc completion bash > /etc/bash_completion.d/oc_bash_completion
```

Comment:
This does not work as there's no sudo rights after a redirect.

```suggestion
sudo sh -c "oc completion bash > /etc/bash_completion.d/oc"
```

Reply:
I see. `oc completion bash | sudo tee /etc/bash_completion.d/oc_bash_completion` might work too.